samples/sid_end_device/Kconfig.sysbuild

```diff
@@ -44,11 +44,23 @@ config NETCORE_APP_UPDATE
 config DFU_MULTI_IMAGE_PACKAGE_NET
 	default y
 
-endif # BOOTLOADER_MCUBOOT
-
 endif # (BOARD_NRF5340DK_NRF5340_CPUAPP || BOARD_THINGY53_NRF5340_CPUAPP)
 
+if BOARD_NRF54L15PDK_NRF54L15_CPUAPP_NS
+
+# Disable checking the external drivers for NS build.
+config PM_OVERRIDE_EXTERNAL_DRIVER_CHECK
+	default y
+
+# override the mcuboot pad size, because it is not set globally for NS build.
+config PM_MCUBOOT_PAD
```

Comment:
I see this config needed for nrf54l15dk also - maybe it should be outside if _ns board? As I understand we always use mcuboot pad 0x800 (set in pm static)